packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuAnchor.tsx

```diff
@@ -74,7 +74,7 @@ export class ContextualMenuAnchor extends ContextualMenuItemWrapper {
                 item={item}
                 classNames={classNames}
                 index={index}
-                onCheckmarkClick={hasCheckmarks && onItemClick ? onItemClick.bind(this, item) : undefined}
+                onCheckmarkClick={hasCheckmarks && onItemClick ? onItemClick : undefined}
```

Comment:
We definitely can't do this, you have to pass the item through. Something like adding a lambda as a pass through in the base might work.

Reply:
Are you sure? See https://github.com/OfficeDev/office-ui-fabric-react/pull/7147#issuecomment-440459693

Reply:
If its because we'd lose `this` within 

https://github.com/OfficeDev/office-ui-fabric-react/blob/92ba8325a3f01c050da0d9e50effb1c399e39663/packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuAnchor.tsx#L97

I'm unsure if that's a concern since we aren't invoking that but rather the prop callback directly.

Reply:
Resolved offline, see:

- https://github.com/OfficeDev/office-ui-fabric-react/pull/7147#issuecomment-440471589
- https://github.com/OfficeDev/office-ui-fabric-react/pull/7147#issuecomment-440473971